packages/core/stories/tutorials/getting-started/angular.ts

```diff
@@ -0,0 +1,111 @@
+import marked from "marked";
+
+export const angularTutorial = marked(`
+# Carbon Charts - Angular
+The Carbon Charts Angular library provides a collection of reusable charting components to build websites and user interfaces. Adopting the library enables developers to use consistent markup, styles, and behavior in prototype and production work.
+
+&nbsp;
+## Install
+
+**with yarn:**
+\`\`\`bash
+yarn add @carbon/charts @carbon/charts-angular d3
+\`\`\`
+
+**with npm:**
+\`\`\`bash
+npm install --save @carbon/charts @carbon/charts-angular d3
+\`\`\`
+
+&nbsp;
+## Getting started
+
+To start using the \`BarChart\` component, try the example below:
+&nbsp;
+
+###### app.module.ts
+\`\`\`ts
+import { NgModule } from "@angular/core";
+import { BrowserModule } from "@angular/platform-browser";
+
+import { ChartsModule } from "@carbon/charts-angular";
+
+import { AppComponent } from "./app.component";
+
+@NgModule({
+  imports: [BrowserModule, ChartsModule],
+  declarations: [AppComponent],
+  bootstrap: [AppComponent],
+})
+export class AppModule {}
+\`\`\`
+
+
+###### app.component.html
+\`\`\`html
+<ibm-stacked-bar-chart
+  class="n-chart"
+  [data]="stackedBarData"
+  [options]="stackedBarOptions"
+  #stackedBarChart
+></ibm-bar-chart>
+\`\`\`
+
+###### app.component.ts
+\`\`\`ts
+import { Component } from "@angular/core";
+
+@Component({
+  selector: "my-app",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.css"],
+})
+export class AppComponent {
+  stackedBarData = {
+    // ...see next section
+  };
+
```

Comment:
same as above for all the other packages^^

Reply:
should i just leave it empty?
` stackedBarData = [];`
or should i say to see the tabular format tutorial or something?

Reply:
yes point to the tabular format